unlock-app/src/components/content/EventContent.tsx

```diff
@@ -0,0 +1,201 @@
+  }
+
+  if (!metadata?.attributes) {
+    return <p>Not an event!</p>
+  }
+
+  const eventData = formatEventData(metadata)
+
+  const addToCalendar = () => {
+    alert('AA')
+  }
+
+  const shareOnTwitter = () => {
+    alert('TT')
+  }
+
+  return (
+    <main className="grid md:grid-cols-[minmax(0,_1fr)_300px] gap-8 mt-8">
+      <section className="">
+        <h1 className="text-5xl md:text-7xl font-bold mb-4">
+          {eventData.title}
+        </h1>
+        <p className="flex flex-rows gap-2 mb-4">
+          <span className="text-brand-gray">Ticket contract</span>
+          <AddressLink
+            lockAddress={lockAddress}
+            network={network}
+          ></AddressLink>
+        </p>
+        <ul
+          className="bold text-xl md:text-2xl mb-6"
+          style={{ color: eventData.accentColor }}
+        >
+          <li className="mb-2">
+            <FaCalendar className="inline mr-2" />
+            {eventData.date}
+          </li>
+          <li className="mb-2">
+            <FaClock className="inline mr-2" />
+            {eventData.time}
+          </li>
+          <li className="mb-2">
+            <Link
+              target="_blank"
+              href={`https://www.google.com/maps/search/?api=1&query=${eventData.address}`}
+            >
+              <GoLocation className="inline mr-2" />
+              {eventData.address}
+            </Link>
+          </li>
+        </ul>
+        <div className="markdown">
+          {/* eslint-disable-next-line react/no-children-prop */}
+          <ReactMarkdown children={eventData.description}></ReactMarkdown>
+        </div>
+      </section>
+      <section className="flex flex flex-col items-center">
+        <img alt={eventData.title} className="mb-5" src={eventData.image} />
+        <ul className="flex justify-around w-1/2">
+          <li className="bg-gray-200 rounded-full">
+            <Tooltip
+              label="Add to Calendar"
+              tip="Add to Calendar"
+              side="bottom"
+            >
+              <button
+                onClick={addToCalendar}
+                className="w-12 h-12 flex justify-center items-center"
+              >
+                <FaRegCalendarPlus className="w-6 h-6" />
+              </button>
+            </Tooltip>
+          </li>
+          <li className="bg-gray-200 rounded-full">
+            <Tooltip
+              label="Share on Twitter"
+              tip="Share on Twitter"
+              side="bottom"
+            >
+              <button
+                onClick={shareOnTwitter}
+                className="w-12 h-12 flex justify-center items-center"
+              >
+                <FiTwitter className="w-6 h-6" />
+              </button>
+            </Tooltip>
+          </li>
+        </ul>
+      </section>
+      <section className="mb-8">
+        {account && (
+          <Button
+            className="mt-8 w-full"
+            style={{ backgroundColor: eventData.accentColor }}
```

Comment:
Also need to set appropriate text color otherwise this would result in an inaccessible button text (eg, accent color is white and text is also white)

Reply:
Do you know of a lib that we would use and would generate the highest contrast for any color (between black and white) since users can select any accent color?

Reply:
https://www.npmjs.com/package/font-color-contrast

Reply:
Ho nice!